analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DoNotUseOutRefParameters.cs

```diff
@@ -102,8 +102,24 @@ public Person(string fname, string mname, string lname)
             lname = LastName;
         }
 
+        public void Deconstruct(out string fname, out string lname, string notAnOutParam) // in Deconstruct all parameters have to be out parameteres.
```

Comment:
`// in Deconstruct all parameters have to be out parameteres.` we no longer check this.